ansible-scylla-monitoring/tasks/common.yml

```diff
@@ -73,11 +73,30 @@
 - set_fact:
     base_dir: "{{ scylla_monitoring_deploy_path }}/{{ unpacked_files.files[0] }}"
 
-- name: insert scylla_servers.yml
+- stat:
+    path: "{{ scylla_servers_yml_path }}"
+  register: scylla_servers_stat
+  delegate_to: localhost
+  run_once: true
+
+- name: insert scylla_servers.yml if provided by the user
   copy:
     src: "{{ scylla_servers_yml_path }}"
     dest: "{{ scylla_monitoring_config_path }}/scylla_servers.yml"
     mode: '0644'
+  when: scylla_servers_stat.stat.exists|bool
+
+- name: generate scylla_servers.yml for monitoring
+  block:
+    - command: nodetool status
+      register: nodetool_status_out
+      delegate_to: "{{ groups['scylla'][0] }}"
```

Comment:
actually one improvement for future would be to pickup a random node (or if you provide an index, then index of that node, since if above node is down, monitoring role won't get the status)